simapp/test_util.go

```diff
@@ -21,3 +23,12 @@ func NewSimAppUNSAFE(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLat
 	gapp = NewSimApp(logger, db, traceStore, loadLatest, invCheckPeriod, baseAppOptions...)
 	return gapp, gapp.keyMain, gapp.keyStaking, gapp.stakingKeeper
 }
+
+func mustMarshalJSONIndent(cdc *codec.Codec, o interface{}) []byte {
```

Comment:
This is nice and shoud live in the `tests` package

Reply:
We can just include this in the codec package. No need for a test.

Reply:
If this was a test utility, it should live in `tests/util.go`. If it isn't, then `codec` is fine